sdk/communication/communication-rooms/samples/v1-beta/javascript/roomOperations.js

```diff
@@ -71,6 +71,15 @@ async function main() {
 
   console.log(`Deleting room with id: ${roomId}...`);
```

Comment:
Can we remove the V1-beta now that we have v1? I see that is what most GAed ACS SDKs are doing (identity, chat, phonenumbers, etc)

Reply:
removed